src/themes/blueberry/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { theme as defaultTheme } from '../../layout'
+import ExampleTypeConstructor from '../../exampleTypes/'
+
+import Input from './Input'
+import Checkbox from './Checkbox'
+import Fonts from './Fonts'
+import Style from './Style'
+import Table from './Table'
+import Button from './Button'
+import ButtonRadio from './ButtonRadio'
+import ListGroupItem from './ListGroupItem'
+import Wrap from './Wrap'
+import TagComponent from './TagComponent'
+import PagerItem from './PagerItem'
+import Adder from './Adder'
+
+export let Pager = props => <ExampleTypes.ResultPager Item={PagerItem} {...props} />
+
+let baseTheme = {
+  ...defaultTheme,
+  Adder,
+  'Adder.Item': ListGroupItem,
+  Wrap,
+  Fonts,
+  Style,
+  Table,
+  Button,
+  Input,
+  Checkbox,
+  RadioList: ButtonRadio,
+  'FieldPicker.Item': ListGroupItem,
+  'TagsInput.TagComponent': TagComponent,
+  Pager,
+}
+
+let { TypeMap, ...ExampleTypes } = ExampleTypeConstructor(baseTheme)
```

Comment:
Note that the example types are still using regular component props without any new theme stuff - this works because `ExampleTypeConstructor` (the `src/exampleTypes` default export) effectively takes a theme object as its argument anyway. 

`exampleTypes` still needs the same treatment as `layout`, but it wasn't necessary for this PR